packages/ocaml-src/ocaml-src.4.13.0/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "2.0"
+maintainer: "opam-devel@lists.ocaml.org"
+authors: "OCaml contributors"
+homepage: "http://ocaml.org/"
+install: ["cp" "-r" "." "%{lib}%/ocaml-src"]
+synopsis: "Compiler sources"
+depends: [
+  "ocaml" {= "4.13.0"}
+]
+extra-files: ["META" "md5=a86cbc1fb202e11ff24cde77754c9d93"]
+url {
+  # TODO: Update on each alpha/beta/rc releases and one last time for the final release
+  src: "https://github.com/ocaml/ocaml/archive/trunk.tar.gz"
```

Comment:
This should be a `git` src so that `opam update` will pick up on upstream commits and track them correctly.

Reply:
if we do that, we need to do the same thing in `ocaml-variants.4.13.0+trunk` as it is not currently the case

Reply:
Those are compiler packages, whereas this is a library package (that is, used as a dependency in various Mirage builds).

Reply:
(note that it's fine to change the compiler packages too, but that's a separate change unrelated to this one)